lib/liquid/extensions.rb

```diff
@@ -25,6 +26,12 @@ def to_liquid
   end
 end
 
+class BigDecimal
+  def to_liquid
+    to_f
```

Comment:
This basically means that we shouldn't store big decimals in liquid.  I don't think we should do that, and it doesn't seem related to adding a filter.